lib/structures/Interaction.js

```diff
@@ -0,0 +1,328 @@
+"use strict";
+
+const Base = require("./Base");
+const Message = require("./Message");
+const Member = require("./Member");
+const User = require("./User");
+const Constants = require("../Constants");
+
+/**
+* Represents an interaction
+* @prop {String} applicationID The id of the interaction's application
+* @prop {String?} channelID The interaction token
+* @prop {Object?} data The data attached to the interaction
+* @prop {Number?} data.componentType The type of Message Component (Message Component only)
+* @prop {String?} data.custom_id The ID of the Button (Message Component)
+* @prop {String?} data.id The ID of the Slash Command (Slash Command)
+* @prop {String?} data.name The command name (Slash Command only)
+* @prop {Array<Object>?} data.options The run Slash Command options (Slash Command only)
+* @prop {String} data.options.name The name of the Slash Command option
+* @prop {Number} data.options.type Command option type, 1-9
+* @prop {Number?} data.options.value The value of the run Slash Command (Mutually exclusive with options)
+* @prop {Array<Object>?} data.options.options The run Slash Command options (Mutually exclusive with value)
+* @prop {String?} guildID The id of the guild in which the interaction was created
+* @prop {String} id The interaction id
+* @prop {Member?} member the member who triggered the interaction (This is only sene when the interaction is invoked within a guild)
```

Comment:
```suggestion
* @prop {Member?} member The member who triggered the interaction (This is only sent when the interaction is invoked within a guild)
```
A typo? Should be `sent`